bedrock/pocket/templates/pocket/privacy.html

```diff
@@ -12,131 +12,129 @@
+      <h2>{{ ftl('pocket-privacy-phishing') }}</h2>
+      <p>{{ ftl('pocket-privacy-identity-theft') }}</p>
     </section>
 
     <section id="manage">
-      <h2>Managing my privacy and email settings</h2>
-      <p>You can access, correct, or delete your personal information and export your data from your Pocket Profile. You can also view and control which third party services, if any, you want to connect to your Pocket account. These settings are at: <a href="https://getpocket.com/options">https://getpocket.com/options</a></p>
-      <p>As part of our default settings to share saved and recommended content, we send emails to our users. You may opt-out of receiving these, and updates on our service and your use, by using the unsubscribe feature on any of our email correspondence or from your account settings. We will try to communicate with you only according to your instructions, although it may take some time to process your request.</p>
+      <h2>{{ ftl('pocket-privacy-managing-my-privacy') }}</h2>
+      <p>{{ ftl('pocket-privacy-you-can-access', pocket_options_url='https://getpocket.com/options' ) }}</p>
+      <p>{{ ftl('pocket-privacy-as-part') }}</p>
     </section>
 
     <section id="policy">
-      <h2>Amendments to policy</h2>
-      <p>Please note this Privacy Policy will change from time to time. We may amend this Privacy Policy at any time by posting the amended terms on our Website that you can access at getpocket.com or through the options or help menu of the Pocket application. All amended terms shall automatically be effective thirty (30) days after they are initially posted on the Website. If we make any material changes to our Privacy Policy that affect user information already stored in our database, we will post a prominent notice on our Website notifying users that our Privacy Policy has changed. Our use of your information shall be governed by the terms and conditions of our Privacy Policy in effect when you first use our application or thirty (30) after the terms are changed, as explained above.</p>
-      <p>If you have questions about any of the provisions described above, please contact us at <a href="mailto:privacy@getpocket.com">privacy@getpocket.com</a> or at the street address listed above.</p>
+      <h2>{{ ftl('pocket-privacy-amendments') }}</h2>
+      <p>{{ ftl('pocket-privacy-please-note', pocket_home_page='getpocket.com') }}</p>
+      <p>{{ ftl('pocket-privacy-if-you-have', pocket__privacy_email_href='mailto:privacy@getpocket.com', pocket__privacy_email='privacy@getpocket.com') }}</p>
```

Comment:
Issue (blocking): variable name typo
```suggestion
      <p>{{ ftl('pocket-privacy-if-you-have', pocket_privacy_email_href='mailto:privacy@getpocket.com', pocket_privacy_email='privacy@getpocket.com') }}</p>
```